website/www/site/content/en/get-started/downloads.md

```diff
@@ -87,16 +87,24 @@ versions denoted `0.x.y`.
 
 ## Releases
 
+### 2.24.0 (2020-07-29)
```

Comment:
Don't forget to update the date before merging.

Reply:
Done.